soh/CMakeLists.txt

```diff
@@ -179,7 +173,7 @@ endif()
 
 # src (decomp) {{{
 file(GLOB_RECURSE src__ RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} "src/*.c" "src/*.h")
-set_source_files_properties(${src__} PROPERTIES COMPILE_OPTIONS ${WARNING_OVERRIDE})
+set_source_files_properties(${src__} PROPERTIES COMPILE_OPTIONS "${WARNING_OVERRIDE}")
```

Comment:
Actually, not only were they necessary there, they were necessary for the LUS one as well, so that if the flag isn't set, it still sets properties with a `""` instead of no argument, which causes configuration errors.